packages/vscode/scripts/build-node.js

```diff
@@ -15,7 +15,7 @@ require('esbuild')
 		bundle: true,
 		sourcemap: isDev ? true : false,
 		outdir: './dist/node',
-		external: ['vscode'],
+		external: ['vscode', './TSXWorker'],
```

Comment:
TODO: Package this correctly, the worker needs to be a separate file so it's not included in the bundle.